src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -1277,7 +1309,8 @@ spinel_tid_t RadioSpinel::GetNextTid(void)
     return tid;
 }
 
-otError RadioSpinel::SendReset(void)
+template <typename InterfaceType, typename ProcessContextType>
+otError RadioSpinel<InterfaceType, ProcessContextType>::SendReset(void)
```

Comment:
There is a `sleep(0)` at the end of this function which is only available via `unistd.h`. It's fine for such minor portability issue since the users can just use a feature flag to provide similar functionality. Just curious whether it's a must to sleep(0) here especially given that the code is not run concurrently. I've tried to remove this line and it looks just fine. @bukepo

Reply:
why `sleep(0)`?

Reply:
I'm not sure why either since this part of the code is not implemented by me. Maybe @bukepo can shed some light on it.

Reply:
This was just trying to let rcp (forkpty based) to run first. And I think we can safely remove it if it is causing portability issues.